hq/app/controllers/SecurityGroupsController.scala

```diff
@@ -26,7 +26,7 @@ class SecurityGroupsController(val config: Configuration, cacheService: CacheSer
     Ok(views.html.sgs.sgs(sortedFlaggedSgs))
   }
 
-  def securityGroupsAccount(accountId: String) = authAction.async {
+  def securityGroupsAccount(accountId: String): Action[AnyContent] = authAction.async {
```

Comment:
I know IntelliJ moans about annotating these but it's wrong. It's an action, we know what it does, fine to omit the annotation (particularly since it isn't very).